packages/selecto/src/SelectoManager.tsx

```diff
@@ -234,7 +237,7 @@ class Selecto extends EventEmitter<SelectoEvents> {
     /**
      * You can get the currently selected targets.
      */
-    public getSelectedTargets(): Array<HTMLElement | SVGElement> {
+    public getSelectedTargets() {
         return this.selectedTargets;
     }
```

Comment:
```ts
const selectedTargets: ElementType[] = [];
```
I removed it because the enhancement code at the top will be type inference.

Reply:
@ssi02014 

If the return type exists, try to insert it into the jsdoc document. I just wish it could be restored to its original code.

Reply:
@daybrush 
Thank you for your review! 🙏
I modified that code again, as shown below. 
```ts
public getSelectedTargets(): ElementType[] {
  return this.selectedTargets;
}
```